server/blob/blob-cassandra/src/test/java/org/apache/james/blob/cassandra/cache/CachedBlobStoreTest.java

```diff
@@ -274,4 +279,37 @@ public void shouldNotCacheWhenReadWithBigStream() {
             assertThat(Mono.from(cache.read(blobId)).blockOptional()).isEmpty();
         });
     }
+
+    @Test
+    public void readBlobstoreCachedShouldPublishTimerMetrics() {
```

Comment:
What happens with the metrics when we try to read a blob that do not exist?

Reply:
It wont run into the cached, could we consider it a `missCache`?

Reply:
No because there is not reasons to cache it.

The main class behaviour is likely the right one however I did not see tests related to that one (and I think we should test this eventuality)